src/app/terms/page.js

```diff
@@ -0,0 +1,94 @@
+"use client";
+/* eslint-disable react/no-unescaped-entities */
+
+import { Box, Typography } from '@mui/material';
+
+
+
+export default function TermsAndConditionsPage() {
+  return (
+    <Box sx={{
```

Comment:
Why Box? Any other page uses container as a top level wrapper. Also you are supposed to know MUI layout components.